index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Proclaim Conference</title>
+  </head>
+  <body>
+    <section class="container">
+      <header class="header">
```

Comment:
- [ ] On mobile the webpage still doesn't fill the entire width. Kindly fix this.
![ice_screenshot_20210804-113440](https://user-images.githubusercontent.com/33323077/128167889-f230aa0b-7fa8-40d8-9b08-0135dbebd1a5.png)